### PR TITLE
Describe the effect of disable_active_migration more accurately

### PR DESCRIPTION
**Pull request by Yanmei-Liu**
Solving issue #343

### CODE REVIEW

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -315,11 +315,14 @@ Path ID 0.
...
-multipath support. They can also start using multiple paths, unless both
-server preferred addresses and a disable_active_migration transport parameter
-were provided by the server, in which case a client is forbidden to establish
-new paths until "after a client has acted on a preferred_address transport
-parameter" ({{Section 18.2. of QUIC-TRANSPORT}}).
+multipath support. They can also start using multiple paths when both endpoints
+have issued available connection IDs for at least one unused Path ID.
+If endpoint receives a disable_active_migration transport parameter
+provided by the peer, it is forbidden to use a new local address
+to establish new paths for the peer address used during handshake,
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:322]:
```suggestion
to establish new paths to the same peer address as used during handshake,
```

**Comment by mirjak** [comment]
This PR is fine but I propose to not merge it now but have some discussion at the meeting first. The use case for this as explained in issue #343 is anycast but I would like to double-check that there are no other reasons that would limit the opening of new paths before the preferred server address is used.

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -315,11 +315,14 @@ Path ID 0.
...
 
 After completing the handshake, endpoints have agreed to enable
-multipath support. They can also start using multiple paths, unless both
-server preferred addresses and a disable_active_migration transport parameter
-were provided by the server, in which case a client is forbidden to establish
-new paths until "after a client has acted on a preferred_address transport
-parameter" ({{Section 18.2. of QUIC-TRANSPORT}}).
+multipath support. They can also start using multiple paths when both endpoints
+have issued available connection IDs for at least one unused Path ID.
+If endpoint receives a disable_active_migration transport parameter
```
Comment (MikeBishop) [inline draft-ietf-quic-multipath.md:320]:
```suggestion
If an endpoint receives a disable_active_migration transport parameter
```

File: draft-ietf-quic-multipath.md [hunk 3]
```diff
@@ -315,11 +315,14 @@ Path ID 0.
...
+provided by the peer, it is forbidden to use a new local address
+to establish new paths to the same peer address as used during handshake,
+but establishment of additional paths to other peer addresses
```
Comment (MikeBishop) [inline draft-ietf-quic-multipath.md:323]:
```suggestion
to establish new paths to the peer's handshake address. However,
establishment of additional paths from any local address to other peer addresses
```

I think we need to state that this TP also doesn't restrict the endpoint to a single local address.

**Review by MikeBishop — CHANGES_REQUESTED** [review]

**Pull request merged**